src/Components/filterBar.js

```diff
@@ -1,17 +1,22 @@
 import React from 'react';
 import './ComponentsCSS/filterBar.scss';
+import { connect } from 'react-redux';
 import { ReactComponent as LogoProfile } from './pictos/profile.svg';
 import { ReactComponent as LogoFilter } from './pictos/filter.svg';
 import { ReactComponent as LogoAdd } from './pictos/plus-full.svg';
 
-const FilterBar = () => (
+const mapStateToProps = state => ({
+  filterKeywordPageDisplay: state.filterKeywordPageDisplay,
+});
+
+const FilterBar = ({ dispatch, filterKeywordPageDisplay }) => (
   <div>
     <div className="footer-box">
-      <LogoFilter className="filter-logo" />
+      <LogoFilter className="filter-logo" onClick={() => dispatch({ type: 'DISPLAY_FILTER_PAGE', filterKeywordPageDisplay: !filterKeywordPageDisplay })} />
```

Comment:
You should change the ```filterKeywordPageDisplay``` on ```!filterKeywordPageDisplay``` in the reducer

Reply:
Gotcha, changed it to a permanent falsen and suppressed the corresponding actions in the component

Reply:
It's corrected, deleted the new dead code in the corresponding component aswell